src/components/Settings.jsx

```diff
@@ -1,11 +1,17 @@
-import React from 'react'
+import React, { useState } from 'react'
 import * as rb from 'react-bootstrap'
 import Sprite from './Sprite'
 import PageTitle from './PageTitle'
+import Seedphrase from './Seedphrase'
+import ToggleSwitch from './ToggleSwitch'
 import { useSettings, useSettingsDispatch } from '../context/SettingsContext'
 import { SATS, BTC } from '../utils'
+import * as Api from '../libs/JmWalletApi'
 
 export default function Settings({ currentWallet }) {
+  const [seed, setSeed] = useState('')
+  const [showSeed, setShowSeed] = useState(false)
```

Comment:
```suggestion
  const [showingSeed, setShowingSeed] = useState(false)
```

Or something similar that makes it immediately clear what this flag does.

Reply:
Ok I dont see the difference between these 2 personally but I will update to use `showingSeed`. :)